1-exercises/B-while-loop/exercise.js

```diff
@@ -1,12 +1,21 @@
-/*
-    while loops can be useful when you want to execute some code as long as some condition is true.    
+// /*
+//     while loops can be useful when you want to execute some code as long as some condition is true.
 
-    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-seperated string.
-    The list of numbers should start with 0. n is being passed in as a parameter.
-*/
+//     Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-separated string.
+//     The list of numbers should start with 0. n is being passed in as a parameter.
+// */
 
 function evenNumbers(n) {
```

Comment:
Since you like operations over collections, how about this :)

```javascript
function evenNumbers(n) {
  console.log([...Array(n).keys()].map(x => x * 2).join(','))
}
```